src/Tribe/Aggregator/Cron.php

```diff
@@ -385,6 +386,75 @@ public function verify_child_record_creation() {
 		}
 	}
 
+	public function start_batch_pushing_records() {
```

Comment:
:no_entry_sign: Missing doc comment for function start_batch_pushing_records()

<sub>hash: 0b5f63b903bcaad8f878685364f168bb</sub>